src/sql/src/plan/statement/ddl.rs

```diff
@@ -4136,10 +4136,15 @@ fn plan_drop_role(
                 for (member_id, grantor_id) in role.membership() {
                     if &id == grantor_id {
                         let member_role = scx.catalog.get_role(member_id);
-                        sql_bail!(
-                            "cannot drop role {}: still depended up by membership of role {} in role {}",
-                            name.as_str(), role.name(), member_role.name()
-                        );
+                        return Err(PlanError::DependentObjectsStillExist {
+                            object_type: "role".to_string(),
+                            object_name: name.as_str().to_string(),
+                            dependents: vec![format!(
+                                "membership of role {} in role {}",
+                                role.name(),
+                                member_role.name()
+                            )],
+                        });
```

Comment:
can you add a test case for this?

Reply:
Actually, I just realized that there is no `CASCADE` for `DROP ROLE` so I'm reverting this line.